optimum/exporters/tasks.py

```diff
@@ -659,6 +661,37 @@ def determine_framework(model: str, framework: str = None) -> str:
 
         return framework
 
+    @staticmethod
+    def infer_task_from_model(model_name_or_path):
+        tasks_to_automodels = {}
+        class_name_prefix = ""
+        if is_torch_available():
+            tasks_to_automodels = TasksManager._TASKS_TO_AUTOMODELS
+        else:
+            tasks_to_automodels = TasksManager._TASKS_TO_TF_AUTOMODELS
+            class_name_prefix = "TF"
+
+        inferred_task_name = None
+        is_local = os.path.isdir(model_name_or_path)
+
+        if is_local:
+            # TODO: implement this.
+            raise NotImplementedError("Cannot infer the task from a local directory yet.")
```

Comment:
not sure we even want to support this, in a first version at least

Reply:
Alright, changed the message and the exception to something that does not open the door for future support (even though it might come someday).